api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -64,10 +64,10 @@ type VirtualMachineImageSpec struct {
 
 // VirtualMachineImageStatus defines the observed state of VirtualMachineImage
 type VirtualMachineImageStatus struct {
-	Uuid        string `json:"uuid,omitempty"`
-	InternalId  string `json:"internalId"`
-	PowerState  string `json:"powerState,omitempty"`
-	SupportedOS *bool  `json:"supportedOS,omitempty"`
+	Uuid          string `json:"uuid,omitempty"`
+	InternalId    string `json:"internalId"`
+	PowerState    string `json:"powerState,omitempty"`
+	GOSCSupported *bool  `json:"goscSupported,omitempty"`
```

Comment:
Could you add some comments for the fields here inside VirtualMaciineImageStatus (the other ones three are pretty self explanatory but lets add them for completeness), mainly GOSCSupported will definitely need some description since it may not be clear to a lot of people.